app/section/sections/connectivity-and-resolution/ui.test.js

```diff
@@ -0,0 +1,205 @@
+import nock from 'nock';
+import { Selector, ClientFunction } from 'testcafe';
+import { ManifestProvider } from '../../../manifestProvider';
+import dashboardWithCompleteSections from '../../../../fixtures/dashboardWithCompleteSections.json';
+
+const connectivityAndResolutionMarketingData = {
+  'minimum-connection-speed': '1Mbps',
+  'minimum-desktop-resolution': '16:9 - 2048 x 1152',
+};
+
+const mocks = (withMarketingData) => {
+  if (withMarketingData) {
+    nock('http://localhost:8080')
+      .get('/api/v1/Solutions/S100000-001/sections/connectivity-and-resolution')
+      .reply(200, connectivityAndResolutionMarketingData);
+  } else {
+    nock('http://localhost:8080')
+      .get('/api/v1/Solutions/S100000-001/sections/connectivity-and-resolution')
+      .reply(200, {});
+  }
+};
+
+const pageSetup = async (t, withMarketingData = false) => {
+  mocks(withMarketingData);
+  await t.navigateTo('http://localhost:1234/S100000-001/section/connectivity-and-resolution');
+};
+
+fixture('Show Connectivity And Resolution page')
+  .afterEach(async (t) => {
+    const isDone = nock.isDone();
+    if (!isDone) {
+      nock.cleanAll();
+    }
+
+    await t.expect(isDone).ok('Not all nock interceptors were used!');
+  });
+
+test('should render the Connectivity And Resolution page title', async (t) => {
+  pageSetup(t);
+
+  const title = Selector('[data-test-id="section-title"]');
+
+  await t
+    .expect(title.innerText).eql('Browser based - connectivity and resolution');
+});
+
+test('should render main advice of section', async (t) => {
+  pageSetup(t);
+
+  const mainAdvice = Selector('[data-test-id="section-main-advice"]');
+
+  await t
+    .expect(mainAdvice.innerText).eql('Answer the following questions about your browser based Solution');
+});
+
+test('should render all the advice of the section', async (t) => {
+  pageSetup(t);
+
+  const sectionManifest = new ManifestProvider().getSectionManifest('connectivity-and-resolution');
+  const expectedAdditionalAdvice = sectionManifest.additionalAdvice.join('\n\n');
+
+  const additionalAdvice = Selector('[data-test-id="section-additional-advice"]');
+
+  await t
+    .expect(additionalAdvice.innerText).eql(expectedAdditionalAdvice);
+});
+
+test('should render the minimum connection speed question', async (t) => {
+  pageSetup(t);
+
+  const minimumConnectionSpeedQuestion = Selector('[data-test-id="combobox-options-minimum-connection-speed"]');
+
+  await t
+    .expect(minimumConnectionSpeedQuestion.find('.nhsuk-label').innerText).eql('Select the minimum connection speed required for your Solution to function?*')
+    .expect(minimumConnectionSpeedQuestion.find('.nhsuk-hint').innerText).eql('Please select below the minimum bandwidth requirement.')
```

Comment:
can these strings come form the manifest?

Reply:
yeah they can. Would it be ok to do that as part of a separate ticket? I rather have a consistent way across the UI tests rather than a mixture of approaches

Reply:
👍 make a tech debt ticket?

Reply:
Tech debt ticket created https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_workitems/edit/3963